docs/topics/streamfield.md

```diff
@@ -8,6 +8,8 @@ For further background on StreamField, and why you would use it instead of a ric
 
 StreamField also offers a rich API to define your own block types, ranging from simple collections of sub-blocks (such as a 'person' block consisting of first name, surname and photograph) to completely custom components with their own editing interface. Within the database, the StreamField content is stored as JSON, ensuring that the full informational content of the field is preserved, rather than just an HTML representation of it.
 
+For more information on how `StreamField` interacts with Django's migration system, and a guide to migrating rich text to `StreamFields`, see the [StreamField migrations page](streamfield_migrations).
```

Comment:
This should probably go at the end of the page. We want to present information in the order that users need to learn it - at this point in the document, when they're only just learning about what a StreamField is, mentioning "you might to know how to migrate them" is a dangling thread they don't need.